config/theme-tabler.php

```diff
@@ -19,22 +19,10 @@
      * Simple pick a layout and let Backpack decide the best look for it.
      * No extra step is required.
      *
-     * Where should the menu be positioned?
+     * Possible values: horizontal, horizontal_dark, horizontal_overlap, vertical,
+     * vertical_dark, vertical_transparent, right_vertical, right_vertical_dark, right_vertical_transparent
      */
     'layout' => 'horizontal',
-    // horizontal
-    // horizontal_dark
-    // horizontal_overlap
-    // vertical
-    // vertical_dark
-    // vertical_transparent
-    // right_vertical
-    // right_vertical_transparent
```

Comment:
Same feedback for the `right_vertical`, `right_vertical_dark` and `right_vertical_transparent` as for the left verticals... I really disagree with having the topbar fixed. Or having it at all 😀